modules/core/src/main/scala/org/tessellation/infrastructure/aci/StateChannelRunner.scala

```diff
@@ -1,123 +1,2 @@
-    for {
-      instances <- new StateChannelInstanceLoader[F].loadFromClasspath
-      contexts <- instances.keySet.toList.traverse {
-        makeStateChannelContext[F]
-      }.map(_.toMap)
-    } yield make(instances, contexts, stateChannelOutputQueue)
-
-  def make[F[_]: Async](
-    stateChannelInstances: Map[Address, StateChannelInstance[F]],
-    stateChannelContexts: Map[Address, StateChannelContext[F]],
-    stateChannelOutputQueue: Queue[F, StateChannelOutput]
-  ): StateChannelRunner[F] = new StateChannelRunner[F] {
-    val logger = Slf4jLogger.getLogger[F]
-
-    val hypergraphContext = new HypergraphContext[F] {
-
-      def getStateChannelContext(address: Address): F[Option[StateChannelContext[F]]] =
-        stateChannelContexts.get(address).pure[F]
-    }
-
-    def initializeKnownCells: F[Unit] =
-      stateChannelContexts.keySet.toList
-        .traverse(initializeCell(_).handleErrorWith {
-          case StateChannelDoesNotExist(address) =>
-            logger.error(s"Trying to initialize state channel for address=${address.show}, but it does not exist!")
-          case err => logger.error(err)("Unknown error")
-        })
-        .void
-
-    private def getContextAndInstance(address: Address): Option[(StateChannelContext[F], StateChannelInstance[F])] =
-      stateChannelContexts.get(address).flatMap { context =>
-        stateChannelInstances.get(address).map { instance =>
-          (context, instance)
-        }
-      }
-
-    def initializeCell(address: Address): F[Unit] =
-      getContextAndInstance(address).map {
-        case (context, instance) =>
-          def runCell(input: Ω): F[Ω] =
-            instance
-              .makeCell(input, hypergraphContext)
-              .run()
-              .flatMap(_.liftTo[F])
-
-          def publishOutput(output: Ω): F[Unit] =
-            instance.kryoSerializer
-              .serialize(output)
-              .liftTo[F]
-              .flatMap { outputBytes =>
-                val scOutput = StateChannelOutput(instance.address, output, outputBytes)
-                stateChannelOutputQueue.offer(scOutput)
-              }
-
-          val pipeline = context.inputs
-            .through(instance.inputPipe)
-            .evalMap(runCell)
-            .through(instance.outputPipe)
-            .evalTap(publishOutput)
-            .compile
-            .drain
-
-          Spawn[F].start(pipeline).void
-      }.getOrElse(StateChannelDoesNotExist(address).raiseError[F, Unit])
-
-    def routeInput(input: StateChannelInput): OptionT[F, Unit] =
-      for {
-        instance <- stateChannelInstances.get(input.address).toOptionT[F]
-        context <- stateChannelContexts.get(input.address).toOptionT[F]
-
-        _input <- instance.kryoSerializer.deserialize[Ω](input.bytes).toOption.toOptionT[F]
-        _ <- OptionT.liftF(context.enqueueInput(_input))
-      } yield ()
-  }
-
-}
+object StateChannelRunner {}
```

Comment:
Empty object

Reply:
My bad, I was removing part by part and I've forgotten to remove it